app/src/components/GrantDetailsRow.vue

```diff
@@ -0,0 +1,110 @@
+<template>
+  <div class="border-b border-grey-100">
+    <div class="grid grid-cols-4 gap-0">
+      <!-- img -->
+      <div class="col-span-4 md:col-span-2 shadow-light flex">
+        <img class="m-auto" :src="logoURI" />
```

Comment:
`logoURI` is optional so we should probably account for that with some default image

Reply:
I've moved the fallback into the template 🙌